pkgs/native_doc_dartifier/analysis_options.yaml

```diff
@@ -0,0 +1,7 @@
+include: package:dart_flutter_team_lints/analysis_options.yaml
+
+analyzer:
```

Comment:
Nit: we use this in a bunch of other packages:

```
analyzer:
  errors:
    todo: ignore
  language:
    strict-casts: true
    strict-inference: true
    strict-raw-types: true

linter:
  rules:
    - dangling_library_doc_comments
    - prefer_const_declarations
    - prefer_expression_function_bodies
    - prefer_final_in_for_each
    - prefer_final_locals
```

Reply:
I copied from another package so we should update all.